deploy/resources/v0.8.0/addons/pipelinetemplates/s2i-go/s2i-go-build-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: s2i-go-build-and-deploy-pipeline
+  namespace: openshift
+  labels:
+    pipeline.openshift.io/runtime: golang
+spec:
+  params:
+    - name: IMAGE_NAME
+      type: string
+  resources:
+    - name: app-source
+      type: git
+    - name: app-image
+      type: image
+
+  tasks:
+    - name: build
+      taskRef:
+        name: s2i-go
+        kind: ClusterTask
+      resources:
+        inputs:
+          - name: source
+            resource: app-source
+        outputs:
+          - name: image
+            resource: app-image
+      params:
+        - name: TLSVERIFY
+          value: "false"
+
+    - name: deploy
```

Comment:
`runAfter` is missing in the pipelines in this PR so that deploy runs after build

Reply:
👏good catch!

Reply:
Good catch, updated.